C/primitive/elements/primitive.c

```diff
@@ -8,6 +8,113 @@
 
 #define PRIMITIVE_TAG(s) "Simplicity\x1F" "Primitive\x1F" "Elements\x1F" s
 
+/* An enumeration of all the types we need to construct to specify the input and output types of all jets created by 'decodeJet'. */
+enum TypeNamesForJets {
+  one     = 0,
+  two     = 1,
+  word2   = 2,
+  word4   = 3,
+  word8   = 4,
+  word16  = 5,
+  word32  = 6,
+  word64  = 7,
+  word128 = 8,
+  word256 = 9, /* The typename for word256 must be at index 9 */
```

Comment:
Have mallocBoundVars return the offset where word256 is located rather than fixing it to the number 9.

Reply:
Done.